### ISSUE TITLE
Gunicorn[UVLoop]WebWorker doesn’t call worker_int and worker_abort callback

### ISSUE THREAD

**Issue by hynek** [aio-libs/aiohttp#1202]
## Long story short

gunicorn [defines](http://docs.gunicorn.org/en/latest/settings.html#worker-int) two callbacks that are called after receiving SIGINT/SIGQUIT & SIGABRT.   `Gunicorn[UVLoop]WebWorker` call them.
## Expected behaviour

The callbacks get called.
## Actual behaviour

The callbacks don't get called.
## Proposed fix

Call the callbacks like `gunicorn.workers.base.Worker` does:

``` python
    def handle_quit(self, sig, frame):
        self.alive = False
        # worker_int callback
        self.cfg.worker_int(self)
        time.sleep(0.1)
        sys.exit(0)

    def handle_abort(self, sig, frame):
        self.alive = False
        self.cfg.worker_abort(self)
        sys.exit(1)
```

**Comment by 1st1** [comment]
Hi, could you please try this with uvloop 0.5.4?  I've completely reimplemented how signals are handled (now it's exactly the same same as to how asyncio handles them).

**Comment by hynek** [comment]
This too is unrelated since it happens even without uvloop. :)

I don’t think I’ve actually opened an issue about uvloop masking out sigterm which indeed is fixed, thanks!

**Comment by fafhrd91** [comment]
fixed by #1420
